test/Oscoin/Test/Storage/Block/Equivalence.hs

```diff
@@ -102,8 +103,8 @@ type StoresUnderTest c tx s m =
 -- SQLite store to pass through chain selection.
 withStores
     :: forall c a.
-       IsCrypto c
-    => (StoresUnderTest c DummyTx DummySeal IO -> IO a)
+       (IsCrypto c, SQLite.StorableTx c)
```

Comment:
Right, if we use the `InternalStorableTx` suggestion, you can still write this, or simply do a copy-paste for the sake of this module 😉